src/components/ViewCourses.js

```diff
@@ -0,0 +1,75 @@
+import React , { Component } from 'react'
+import { ipcRenderer } from 'electron'
+import { hashHistory } from 'react-router'
+
+export default class ViewCourses extends Component {
+  componentWillMount () {
+    ipcRenderer.send('get-courses')
+    this.setState({
+      courseList: null
+    })
+  }
+  componentDidMount () {
+    ipcRenderer.on('get-courses', (event, arg) => {
+      this.setState({
+        courseList: arg
+      })
+    })
+  }
+
+  renderLoading () {
+    return (
+      <div className='loaderWrapper'>
+        <div className='loader'></div>
+      </div>
+    )
+  }
+
+  renderData () {
+    return (
+      <div className='dashboard'>
+        <div className='tableContainer'>
+          <div className='tableHeader'>
+            <div className='tableColumn courseName'>
+              Name
+            </div>
+            <div className='tableColumn courseDate'>
+              Start Date
+            </div>
+            <div className='tableColumn courseDuration'>
+              Duration
+            </div>
+          </div>
+          <div className='tableBody'>
+               {this.state.courseList.map(course=> {
+                  let date = new Date(course.startDate)
```

Comment:
Created a new method for map function inside component class, can't pull `date` variable outside  the JSX. It is there just to parse start date ISO string into date object